generated/python/gax-google-monitoring-v3/README.rst

```diff
@@ -0,0 +1,78 @@
+GAX library for the Google Monitoring API
```

Comment:
FYI, these hand-edits were not applied to the latest re-gen. No worries, since we have to do 0.10.1, but just wanted to remind you. We'll start automating these next week.